### ISSUE TITLE
Performance gap (~6%) when evaluating original model on VHRRoad dataset

### ISSUE THREAD

**Issue by chihuajiao** [vvangfaye/HoliTracer#13]
Hi, thanks for releasing the pretrained weights.

I am evaluating the released pretrained model on the VHRRoad dataset,  but I observe about 6% lower performance than reported in the paper.

python tools/eval/seg_evaluation.py
pre_path = /HoliTracer-master/dataset/test/Hol_ori
ann_path = /HoliTracer-master/dataset/test/mask
Computing metrics...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 21/21 [00:50<00:00,  2.41s/it]
0.07687914115533903
{'Overall_Accuracy': 0.98181407, 'Precision': 0.549328634, 'Recall': 0.586433619, 'Iou': 0.395941323, 'MIou': 0.688767998, 'F1_score': 0.567275022, 'Boundary_IoU': 0.076879141}

I am wondering whether the performance gap might be caused by a mismatch in the dataset split I am using, or if the released pretrained checkpoint corresponds to a different training/evaluation setup.

**Comment by vvangfaye** [comment]
Hi, do the other two datasets have the same issue?

**Comment by chihuajiao** [comment]
I tested on the WHU building dataset that the differences seem to be small,
Computing metrics...
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 40/40 [00:54<00:00,  1.37s/it]
0.4955043760294155
{'Overall_Accuracy': 0.985894279, 'Precision': 0.955734495, 'Recall': 0.954247644, 'Iou': 0.913858183, 'MIou': 0.94863508, 'F1_score': 0.954990491, 'Boundary_IoU': 0.495504376}

**Comment by vvangfaye** [comment]
I caculate again:

<img width="1722" height="122" alt="Image" src="https://github.com/user-attachments/assets/5718d37a-a1c6-4a39-8976-b21d014af56a" />

How do you get the predict result?
If you use the demo.py file, you need to change the `[1,3,6]` to `[1,5,10]`.

https://github.com/vvangfaye/HoliTracer/blob/f216512616efa7241e0a43917d74b02baf73cc94/demo.py#L91
